pages/maintainer/build_cache.md

```diff
@@ -253,6 +253,9 @@ about SAS tokens.  You can obtain a SAS token via the [Settings > Access keys](
 https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal)
 page for your storage account.
 
+For Amazon S3, this will be your AWS Access Key ID and AWS Secret Access Key separated by a colon,
+like this: `<AccessKeyID>:<SecretAccessKey>`.
```

Comment:
@dtinth thanks for making this fix! We overlooked your PR because the Rush website has been moved to a new Docusaurus monorepo. I've reapplied your fix in this PR in the new repo: https://github.com/microsoft/rushstack-websites/pull/82